controllers/admin/AdminSearchConfController.php

```diff
@@ -472,7 +472,11 @@ public function processSave()
         }
 
         if (empty($this->errors)) {
-            $this->confirmations[] = $this->trans('Creation successful', [], 'Admin.Shopparameters.Notification');
+            if (Tools::getValue('id_alias')) {
+                $this->confirmations[] = $this->trans('Successful update.', [], 'Admin.Notifications.Success');
+            } else {
+                $this->confirmations[] = $this->trans('Successful creation.', [], 'Admin.Notifications.Success');
```

Comment:
```suggestion
                $this->confirmations[] = $this->trans('Successful creation', [], 'Admin.Notifications.Success');
```
This new string (without the period and in Admin.Notifications.Success) was introduced in other PRs so let's stick with this one. :-)